### PR TITLE
Add a dictionary colon rule. Also make it so that the description tha…

### PR DESCRIPTION
**Pull request by codeman9**
…t is printed matches the configuration.

Redo rule on current master.

### CODE REVIEW

File: Tests/SwiftLintFrameworkTests/RulesTests.swift [hunk 1]
```diff
@@ -9,7 +9,8 @@
 import SwiftLintFramework
 import XCTest
 
-// swiftlint:disable:next type_body_length
+// swiftlint:disable type_body_length
```
Comment (jpsim) [inline Tests/SwiftLintFrameworkTests/RulesTests.swift]: This could be: `// swiftlint:disable type_body_length file_length`

Comment (jpsim) [inline Tests/SwiftLintFrameworkTests/RulesTests.swift]: make sure you're using SwiftLint 0.13.2, since support for this syntax was only recently added

Comment (codeman9) [inline Tests/SwiftLintFrameworkTests/RulesTests.swift]: Hmmm...I did a `brew upgrade swiftlint` and it updated to 0.13.2, but I'm still getting an error and a warning when I put this on a single line.

**Comment by jpsim** [comment]
I'm not terribly thrilled about the way this is implemented, but unfortunately can't really give concrete feedback as to why. Maybe another contributor can chime in and help review?

**Comment by codeman9** [comment]
Since my other pull request makes the custom rules correctable, this particular rule as a .swift rule vs a custom regex becomes less valuable for us. In fact, my other pull request #944 uses the regex for this rule in its correctable test to show that this rule isn't as valuable as part of the ColonRule.swift.

**Comment by codeman9** [comment]
I think I'm going to decline/close this PR. I've been doing more work on getting the custom rules to work the way we want and have implemented a regex that matches this need there as well as being correctable. See PR #944

**Pull request closed without merging**